### PR TITLE
[skip ci] Initial breakdown file for Byron-Shelley Bridge

### PR DESCRIPTION
**Pull request by ruhatch**

### CODE REVIEW

File: roadmap/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+- ~~Test PBFT against mock ledger~~
+
+- Test PBFT against real (Byron) ledger
+
+
+### Validate Valid Generated Chains against Full Spec [2]
+
+- Byron ledger rules fully implemented in executable spec - Nick/Luke
+  - ~~Delegation rules~~
+  - UTXO rules
```
Comment (dnadales) [inline roadmap/README.md:74]: I thought the UTxO rules where already implemented :thinking:

File: roadmap/README.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    - Incorporate update rules
```
Comment (dnadales) [inline roadmap/README.md:75]: Do we have an issue to link to this? I'm don't know "incorporate update rules" means in this context....

File: roadmap/README.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+- ~~Test validation rules against mainnet~~
+  - ~~Add update mechanism into mainnet chain tests~~
+
+- Generator for (valid) full chain
+  - ~~Merge Ru’s PR with hedgehog stuff~~
+  - Implement generator for blocks with all stuff
+  - ~~Scaffolding to embed CHAIN inside h-s-m~~
+  - ~~Scaffolding for other systems inside h-s-m~~
+
+- Elaboration of full abstract chain
```
Comment (dnadales) [inline roadmap/README.md:97]: Maybe we need to break this down, and also specify what we already have (delegation + utxo?).

File: roadmap/README.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  - CLOSED: [2019-03-29 Fri 14:17]
+
+- ~~Test PBFT against mock ledger~~
+
+- Test PBFT against real (Byron) ledger
+
+
+### Validate Valid Generated Chains against Full Spec [2]
+
+- Byron ledger rules fully implemented in executable spec - Nick/Luke
```
Comment (dnadales) [inline roadmap/README.md:72]: Shouldn't this be under `### Finish work on Byron executable specs [2]`?

**Review by dnadales — APPROVED** [review]
Thank you Ru!

**Comment by ruhatch** [comment]
I've moved this to the wiki [here](https://github.com/input-output-hk/cardano-ledger/wiki/Roadmap), so feel free to edit it!

**Pull request closed without merging**